### PR TITLE
Set the task due date to the start of the day

### PR DESCRIPTION
**Pull request by garethbowen**
#3097

### CODE REVIEW

File: tests/nodeunit/unit/nootils.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    fields: { last_menstrual_period: 3 }
+  };
+  var actual = nootils.getLmpDate(doc);
+  test.equal(actual.toISOString(), '2017-01-09T00:00:00.000Z');
+  test.done();
+};
+
+exports['isTimely returns false if too early'] = function(test) {
+  var yesterday = new Date();
+  yesterday.setDate(yesterday.getDate() + 1);
```
Comment (estellecomment) [inline tests/nodeunit/unit/nootils.js]: erm that looks more like `tomorrow`, in universes with unidirectional time

File: static/js/modules/nootils.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ module.exports = function(settings) {
   return {
     isTimely: function(date, event) {
       var due = new Date(date);
-      var start = this.addDate(null, event.start);
-      var end = this.addDate(null, (event.end + 1) * -1);
+      var start = new Date();
+      start.setUTCDate(start.getUTCDate() + event.start);
```
Comment (estellecomment) [inline static/js/modules/nootils.js]: can you reuse `addDate` here, and below? Or is that not what's going on?
(these date manipulations are pretty obscure to me)

File: tests/nodeunit/unit/nootils.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  var event = {
+    start: 0,
+    end: 2
+  };
+  var actual = nootils.isTimely(yesterday, event);
+  test.equal(actual, false);
+  test.done();
+};
+
+exports['isTimely returns false if too late'] = function(test) {
```
Comment (estellecomment) [inline tests/nodeunit/unit/nootils.js]: any edge cases for `isTimely` when you use a date with a time of day in it?

File: tests/nodeunit/unit/nootils.js (hunk 1 above)
Comment (garethbowen) [inline tests/nodeunit/unit/nootils.js]: That'll show me for trying to use descriptive variable names...

File: tests/nodeunit/unit/nootils.js (hunk 3 above)
Comment (garethbowen) [inline tests/nodeunit/unit/nootils.js]: None I can think of. `isTimely` should be unchanged from previous implementations, I just had to decouple it from the `addDate` function that I changed.

File: static/js/modules/nootils.js (hunk 2 above)
Comment (garethbowen) [inline static/js/modules/nootils.js]: That's what it was doing before, but I changed how addDate works so decoupled the two functions so the `isTimely` function would work the same as it used to.
